### DISCUSSION TITLE
Multilines for Labels

### DISCUSSION THREAD

**Discussion by DDaehn** [themustafaomar/jsvectormap#58] [Q&A] [answered]
Is there a way to add a multiline Text to the label? 
We want to display several information by hovering over the marker, currently we cant find a way to split it into multiple lines, so its just a very long one line string.

If its not possible currently, I think it would be a great feature to add.

**Comment by themustafaomar** [comment]
Very sorry for the late response, unfortunately, we can't add any HTML attributes or even add a `new line` inside SVG `text` tag https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text, hopefully, I'll consider supporting this feature when I got the time :]

**Comment by ccll** [comment, marked as answer]
Same problem here, and found a workaround to this with some JS DOM magic.
Just find the label elements and replace the innerHTML with multiline `<tspan>` elements:
```ts
const labelGroupElement = document.getElementById('jvm-markers-labels-group')
if (labelGroupElement) {
    for (const child of labelGroupElement.children) {
        const markerName = child.innerHTML
        const location = data.locations.find(
            (marker) => marker.displayLocation === markerName,
        )
        const region = location?.displayRegion
        const x = child.getAttribute('x')
        child.innerHTML = `
            <tspan x="${x}" dy="-0.3em" font-size="16" font-weight="bold">${markerName}</tspan>
            <tspan x="${x}" dy="1.2em" fill="rgb(182 185 191)" font-weight="bold">${region}</tspan>
        `
        const translateX = location?.coordsLabelTranslate[0] ?? '0'
        const translateY = location?.coordsLabelTranslate[1] ?? '0'
        child.setAttribute('transform', `translate(${translateX}, ${translateY})`)
    }
}
```

The result:
![image](https://github.com/themustafaomar/jsvectormap/assets/5366582/1edf1aed-db8c-41dc-9d3a-c546669b10b9)
